exchanges/bybit/bybit_test.go

```diff
@@ -3397,84 +3441,6 @@ func TestConstructOrderDetails(t *testing.T) {
-	}
-	var err error
-	for x := range snapshots {
-		err = b.wsHandleData(x, []byte(snapshots[x]))
-		if err != nil {
-			t.Fatal(err)
-		}
-	}
-
-	// Spot update processing
-	data := `{ "symbol": "BTC-USDT", "lastPrice": "21109.77", "highPrice24h": "21426.99", "lowPrice24h": "20575", "prevPrice24h": "20704.93", "volume24h": "6780.866843", "turnover24h": "141946527.22907118", "price24hPcnt": "0.0196", "usdIndexPrice": "21120.2400136" }`
-	var result WsSpotTicker
-	err = json.Unmarshal([]byte(data), &result)
-	if err != nil {
-		t.Fatal(err)
-	}
-	cp, err := b.ExtractCurrencyPair(result.Symbol, asset.Spot, false)
-	if err != nil {
-		t.Fatal(err)
-	}
-	_, err = b.updateSpotTickerInformation(&result, cp)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	// Linear update processing
-	data = `{ "symbol": "BTC_USDT", "tickDirection": "PlusTick", "price24hPcnt": "0.017103", "lastPrice": "17216.00", "prevPrice24h": "16926.50", "highPrice24h": "17281.50", "lowPrice24h": "16915.00", "prevPrice1h": "17238.00", "markPrice": "17217.33", "indexPrice": "17227.36", "openInterest": "68744.761", "openInterestValue": "1183601235.91", "turnover24h": "1570383121.943499", "volume24h": "91705.276", "nextFundingTime": "1673280000000", "fundingRate": "-0.000212", "bid1Price": "17215.50", "bid1Size": "84.489", "ask1Price": "17216.00", "ask1Size": "83.020" }`
-	var resultLinear WsLinearTicker
-	err = json.Unmarshal([]byte(data), &resultLinear)
-	if err != nil {
-		t.Fatal(err)
-	}
-	cp, err = b.ExtractCurrencyPair(resultLinear.Symbol, asset.USDTMarginedFutures, false)
-	if err != nil {
-		t.Fatal(err)
-	}
-	_, err = b.updateTickerInformation(&resultLinear, cp, asset.USDTMarginedFutures)
-	if err != nil {
-		t.Error(err)
-	}
-
-	// Options update processing
-	data = `{"symbol": "BTC-USD-220930-28000-P", "bidPrice": "0", "bidSize": "0", "bidIv": "0", "askPrice": "10", "askSize": "5.1", "askIv": "0.514", "lastPrice": "10", "highPrice24h": "25", "lowPrice24h": "5", "markPrice": "7.86976724", "indexPrice": "16823.73", "markPriceIv": "0.4896", "underlyingPrice": "16815.1", "openInterest": "49.85", "turnover24h": "446802.8473", "volume24h": "26.55", "totalVolume": "86", "totalTurnover": "1437431", "delta": "0.047831", "gamma": "0.00021453", "vega": "0.81351067", "theta": "-19.9115368", "predictedDeliveryPrice": "0", "change24h": "-0.33333334" }`
```

Comment:
I was sorta nice that we were testing updates for each type instead of just USDTMarginedFutures BTCUSDT

Reply:
I did see that. I think I was looking too hard at the json file when I made the comment

Reply:
Fixed 720061c

Everything is tested now, with real data.